src/component/common/Table/TablePanel/TablePanel.tsx

```diff
@@ -0,0 +1,18 @@
+import { forwardRef, ReactNode } from 'react';
+import { Paper } from '@mui/material';
+import { Box } from '@mui/material';
+import { useStyles } from './TablePanel.styles';
+
+export const TablePanel = forwardRef<
```

Comment:
Maybe `TableContainer` or `TablePaper`?

Reply:
`TableContainer` 👍🏻 . I'll refactor this in next PR.